app/models/concerns/hydra/works/generic_work_behavior.rb

```diff
@@ -0,0 +1,75 @@
+module Hydra::Works
+  module GenericWorkBehavior
+    extend ActiveSupport::Concern
+    # TODO: Extend rdf type to include RDFVocabularies::HydraWorks.GenericWork
+
+    # behavior:
+    #   1) Hydra::Works::GenericWork can NOT aggregate Hydra::PCDM::Collection
+    #   2) Hydra::Works::GenericWork can NOT aggregate Hydra::Works::Collection
+    #   3) Hydra::Works::GenericWork can NOT aggregate Works::GenericWork unless it is also a Hydra::Works::GenericFile
+    #   4) Hydra::Works::GenericWork can aggregate Hydra::Works::GenericFile
+    #   5) Hydra::Works::GenericWork can NOT contain PCDM::File
+    #   6) Hydra::Works::GenericWork can NOT contain Hydra::Works::File
+    #   7) Hydra::Works::GenericWork can NOT aggregate non-PCDM object
+    #   8) Hydra::Works::GenericWork can have descriptive metadata
+    #   9) Hydra::Works::GenericWork can have access metadata
+    # TODO: add code to enforce behavior rules
+
+
+    # TODO: Inherit members as a private method so setting objects on aggregations has to go through the following methods.
+    # TODO: FIX: All of the following methods for aggregations are effected by the error "uninitialized constant Member".
+    #       See collection_spec test for more information.
+    
+    def << arg
```

Comment:
I think we should remove this method for the reasons given in GenericFileBehavior